Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -122,6 +122,12 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
       .observeValues { [weak self] message in
         self?.present(UIAlertController.genericError(message), animated: true)
     }
+
+    self.viewModel.outputs.editButtonTitle
+      .observeForUI()
+      .observeValues { [weak self] title in
+        self?.editButton.title = title
```

Comment:
The documentation mentioned kinda vague fact that the `title` should be set before adding the item to the bar - https://developer.apple.com/documentation/uikit/uibaritem/1616412-title which makes me wonder if we should actually re-create the `UIBarButtonItem` but with a different title?

Something like:
```
private func editButton(_ title: String) -> UIBarButtonItem {
    return UIBarButtonItem(
      title: title,
      style: .plain,
      target: self,
      action: #selector(edit)
    )
  }
```

Reply:
Not sure if this is related to the bug that I've reported on the Trello card 🤷‍♂️

Reply:
Ah interesting, I could try that, thanks 👍

Reply:
I wasn't able to reproduce the truncation issue but I've tried a property called `possibleTitles`, first I've heard of it - https://developer.apple.com/documentation/uikit/uibarbuttonitem/1617132-possibletitles

Let me know if you're still seeing it.